README.md

```diff
@@ -1 +1,90 @@
-# functional-harness
+## testcafe-functional-harness
+
+Runs functional tests in local browsers and on SauceLabs.
+
+#### What it does
+
+1. Starts the server with the test pages.
+2. Opens the specified browsers (locally or on SauceLabs).
+3. Runs mocha tests.
+4. Closes the browsers and stops the server after all tests are finished.
+
```

Comment:
So, regarding the example... It took me some time to figure out what happens in these mutually referencing modules...

The point of entry is `testRunner.run()`, which creates the testCafe and harness instances, then makes the harness run mocha tests, which, in turn, reference back (OMG https://goo.gl/Hh3oe3) the test runner to obtain the very same instance of testCafe that was used to initialize the harness in order to test it.

Rather sophisticated... (or is it just me, again)

Anyway, did I get it right?